spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -205,10 +205,14 @@ Spell Actions (Per Exec Doc):
   * [ ] MKR transfers
     * [ ] Recipient addresses match Exec Doc
     * [ ] Transfer values match Exec Doc
+        * [ ] All values are specified with 2 decimals
+        * [ ] Sum of all values match Exec Doc
```

Comment:
I thought [the agreement was](https://docs.google.com/document/d/18FxXF4aOO-IkmZcmXvZ7Vrw5Qi9PFOaBETmNLu9GNCo/edit#heading=h.ratfmanx8uq9) that from now on, it has to be part of the tests. From the current formulation (and the place where it's added) it sounds like it's the reviewer-only responsibility to put the numbers together. I think it's better to make it part of the code
```suggestion
        * [ ] Sum of all values is separately tested and matches Exec Doc
```

Reply:
Yeah, I was going to PR the spell repo with this specific test. I just refrained from putting it here because there's no mention to specific tests in the entire checklist.

Reply:
Then, let's add this new section [under line 101](https://github.com/makerdao/pe-checklists/blob/2c635a721c83a6f09a942c912856ab70db165e2e/spell/spell-crafter-mainnet-workflow.md?plain=1#L101)? Otherwise, the crafter will always miss it and the reviewer will always have to remind them 🙂

Reply:
Makes sense. Will add it.

Reply:
Done.

Reply:
Do you also want to accept the proposed change to reformulate the reviewer checklist? Or, at least, fix spelling error

```suggestion
        * [ ] Sum of all values matches Exec Doc
```
(here and in the other place)

Reply:
Not sure if it's an error. IIRC `match` could conform to both `sum` (so it would have to be `matches`), but also to `values`, which is closer in the sentence.
That's beyond my depth in English grammar, though. I'll make the changes you suggested.